examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Don't agree at all. Why do I need to put something in my JSX just to render global styles? That doesn't compute at all, no matter if it's declarative or not, because JSX ≈ DOM, not CSSOM.